Libraries/PeriphDrivers/Include/MAX78002/csi2.h

```diff
@@ -350,26 +371,11 @@ int MXC_CSI2_GetLaneCtrlSource(mxc_csi2_lane_src_t *src);
 
 /**
  * @brief      Grab the configured image details.
- * @param      img      Pointer to image buffer.
- * @param      imgLen   Pointer to byte length of image.
- * @param      w        Pointer to Image Width.
- * @param      h        Pointer to Image Height.
- */
-void MXC_CSI2_GetImageDetails(uint8_t **img, uint32_t *imgLen, uint32_t *w, uint32_t *h);
-
-/**
- * @brief      Callback function for CSI2.
- * @param      req        The details of the image capture (holds callback).
- * @param      retVal    Return value for callback function, usually error code.
- * @return     #E_NO_ERROR if everything is successful.
+ * @param[out]      imgLen   Pointer to the total length of the image (in bytes).
+ * @param[out]      w        Pointer to image width (in pixels).
+ * @param[out]      h        Pointer to image height (in pixels).
  */
-int MXC_CSI2_Callback(mxc_csi2_req_t *req, int retVal);
-
-/**
- * @brief      Interrupt Handler Function.
- * @return     #E_NO_ERROR if everything is successful.
- */
-int MXC_CSI2_Handler(void);
```

Comment:
Is the mxc_csi2_line_handler_cb_t supposed to replace this function?

Reply:
No, but I wanted to intentionally remove the publicly exposed handler.  The drivers are completely interrupt driven from `MXC_CSI2_RevAHandler` so there is no option to override the CSI2 IRQ handler (yet).